backend/test/integration/langchain.test.ts

```diff
@@ -115,28 +116,28 @@ jest.mock('@src/openai', () => {
 	};
 });
 
-beforeEach(() => {
-	// reset environment variables
-	process.env = {
-		OPENAI_API_KEY: 'sk-12345',
-	};
-
-	mockFromLLM.mockImplementation(() => mockRetrievalQAChain);
-});
+describe('langchain integration tests ', () => {
+	beforeEach(() => {
+		// reset environment variables
+		process.env = {
+			OPENAI_API_KEY: 'sk-12345',
+		};
+
+		mockFromLLM.mockImplementation(() => mockRetrievalQAChain);
+		mockLoader.mockResolvedValue([]);
+	});
 
-afterEach(() => {
-	mockPromptEvalChain.call.mockRestore();
-	mockRetrievalQAChain.call.mockRestore();
-	mockFromLLM.mockRestore();
-	mockFromTemplate.mockRestore();
-});
+	afterEach(() => {
+		mockPromptEvalChain.call.mockReset();
+		mockRetrievalQAChain.call.mockReset();
+		mockFromLLM.mockReset();
+		mockFromTemplate.mockReset();
+		mockLoader.mockReset();
```

Comment:
could you use `mockClear()` instead and then not have to do `mockLoader.mockResolvedValue([]);` in the beforeEach?

[doc](mockLoader.mockResolvedValue([]);)

Reply:
Done